From 02b658bc3252dac2000ebdb5390b5daf4e59a987 Mon Sep 17 00:00:00 2001
From: Jami Kettunen <jami.kettunen@protonmail.com>
Date: Fri, 3 Sep 2021 20:49:21 +0300
Subject: [PATCH 116/127] [TEMP] oneplus5: disable all speaker amp stuff

We don't want to blow this up while messing around with audio stuff
early on...
---
 arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi
index 0270437704e3..5fbe0b77d5c7 100644
--- a/arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi
@@ -271,7 +271,7 @@ cts {
 };
 
 &blsp2_i2c3 {
-	status = "okay";
+	status = "disabled";
 
 	audio-codec@36 {
 		compatible = "nxp,tfa9890";
@@ -284,6 +284,7 @@ audio-codec@36 {
 
 		sound-name-prefix = "Mono Speaker";
 		#sound-dai-cells = <0>;
+		status = "disabled";
 	};
 };
 
-- 
2.38.1

